app/src/test/java/org/simple/clinic/search/results/PatientSearchResultsControllerTest.kt

```diff
@@ -39,65 +39,39 @@ class PatientSearchResultsControllerTest {
   private val uiEvents = PublishSubject.create<UiEvent>()
```

Comment:
There are some unused imports in this file. Can you clear them?

Reply:
Done.